src/Microsoft.ML.Transforms/NAReplaceTransform.cs

```diff
@@ -165,6 +167,44 @@ private static string TestType<T>(ColumnType type)
             return null;
         }
 
+        public class ColumnInfo
+        {
+            public readonly string Input;
+            public readonly string Output;
+            public readonly bool ImputeBySlot;
+            public readonly ReplacementKind Kind;
+
+            public ColumnInfo(string input, string output, ReplacementKind kind = ReplacementKind.DefaultValue, bool imputeBySlot = true)
```

Comment:
>ColumnInfo [](start = 19, length = 10)

summary comment

Reply:
summary: Describes how the transformer handles one column pair.
input: name of input column
output: name of output column
replacementMode: what to replace the missing value with
imputeBySlot: if true, per-slot imputation of replacement is performed. Otherwise, replacement value is imputed for the entire vector column. This setting is ignored for scalars and variable vectors, where imputation is always for the entire column.

---
In reply to: [217855806](https://github.com/dotnet/machinelearning/pull/917#discussion_r217855806) [](ancestors = 217855806,217834730)